java/java.hints/src/org/netbeans/modules/java/hints/errors/CreateClassFix.java

```diff
@@ -320,9 +320,7 @@ public void run(final WorkingCopy working) throws IOException {
                         default: throw new IllegalStateException();
                     }
                     CompilationUnitTree cut = make.CompilationUnit(targetSourceRoot, packageName.replace('.', '/') + '/' + simpleName + ".java", Collections.<ImportTree>emptyList(), Collections.singletonList(source));
-                    ClassTree nue = createConstructor(working, new TreePath(new TreePath(cut), source));
                     working.rewrite(null, cut);
-                    working.rewrite(source, nue);
```

Comment:
Apologies, I don't quite follow this change - I thought it's desirable to generate a constructor (?). If the constructor is not wanted here, the `implement` method below (also generates one) may need to be modified as well.

Reply:
The language server uses `getModificationResult()` to get changes that need to send back to LSP client as a response to codeAction request. These changes are applied at the client side. On the other hand, `implement` is used by ordinary NetBeans UI and uses createFromTemplate mechanism to create new outer class. But you are right - constructor should be generated even if called via LSP. Fixed.